### PR TITLE
Alvin's question

### PR DESCRIPTION
**Pull request by alvinlim540**
Some update to the question raised by @Zhiyuan-Amos 

- inherit code base or build from scratch?
- deployment for front end/back-end
- Tech stack: framework preferred?

Digital Concierge Questions
A run-thru of the digital concierge process

links of existing solution
CI/CO
Digital Concierge

### CODE REVIEW

**Comment by Zhiyuan-Amos** [comment]
@ahyong92 can you help Alvin with the merge conflict & it seems to be a problem of reusing the same branch

File: docs/to_clarify.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 1. As mentioned under "Context" that "hotels... have tried different digital check-in solutions to limited success", are you able to show us exactly what are some of the solutions that have been tried and in which aspects are they lacking? Having these references would help us to come up with a better solution.
 
+1. Are there systems depending on the existing check-in/out system? If so, what are the APIs used to communicate between these systems? Will documentation & assistance be provided if we face certain issues interfacing with these systems?
+
+1. If there is a need in additional API which is not provided by the existing systems, will they be assistance for us to create the additional APis? Or is there a possiblity where we inherit the codebase and implement the API on our own?
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: > will the~~y~~**re** be assistance

> additional AP~~i~~**I**s

> possib**i**lity

File: docs/to_clarify.md [hunk 2]
```diff
@@ -35,5 +39,40 @@
...
+
+    e. Deployment related: 
+    1. Front-end: Are we supposed to host the app on app store? Or do we need an app, or just web will do?
+    2. Back-end: Do we have to deploy the servers on our own? Or do we write the code and the system admins deploy for us?
+
+    f. Techstack related:
+    1. Any preferred frameworks?
+
+
+
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: Unnecessary new lines

File: docs/to_clarify.md [hunk 3]
```diff
@@ -35,5 +39,40 @@
     a. Which government stipulated system is currently being used, and does the existing system provide API for our solution to communicate with it?
 
     b. Can we have several different samples of the above documents and the details of what are being verified for these documents?
+    
+    c. Are we supposed to inherit the current codebase and integrate or Build a new system from scratch?
+    
+    d.If inheriting their codebase, please provide us the architecture knowledge so that we can write the code in consistent manner.
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: Missing space after d.

File: docs/to_clarify.md [hunk 4]
```diff
@@ -35,5 +39,40 @@
...
+
+    e. Deployment related: 
+    1. Front-end: Are we supposed to host the app on app store? Or do we need an app, or just web will do?
+    2. Back-end: Do we have to deploy the servers on our own? Or do we write the code and the system admins deploy for us?
+
+    f. Techstack related:
+    1. Any preferred frameworks?
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: Content placed in wrong segment

File: docs/to_clarify.md [hunk 5]
```diff
@@ -35,5 +39,40 @@
...
+
+
+
+1. Stated under the section "What Are We Looking For" that the solution should "streamline the back-end allocation of rooms by ensuring room requests are clearly accounted for on the back-end, to ensure rooms and amenities are available upon guest arrival". Is this currently being done, and if so, how is it being done?
+
+### Digital Concierge Questions, If Unanswered
+1. Can you run through with us the current digital concierge process? Please do share with us on the idea how you want it to be integrated. This will help us have a better understanding of the pain points experienced when using the existing solution.
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: Missing new line after header

File: docs/to_clarify.md [hunk 6]
```diff
@@ -35,5 +39,40 @@
...
+
+1. We have looked up some certain latest existing solutions on HotelTechReport. Here is some of the existing solutions that i have narrowed down.
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: > Here ~~is~~ are some

> that ~~i~~**I** have

File: docs/to_clarify.md [hunk 7]
```diff
@@ -35,5 +39,40 @@
     a. Which government stipulated system is currently being used, and does the existing system provide API for our solution to communicate with it?
 
     b. Can we have several different samples of the above documents and the details of what are being verified for these documents?
+    
+    c. Are we supposed to inherit the current codebase and integrate or Build a new system from scratch?
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: > or ~~B~~**b**uild a

**Review by Zhiyuan-Amos — CHANGES_REQUESTED** [review]
Made a brief review. In summary, there are multiple English mistakes in the document. Ideally, we don't want to spend time reviewing each other's English, so put your content through a English checker first before submitting a PR.

Also, there are multiple stylistic issues e.g. unnecessary new lines, missing new line. To ensure that the content is easy to read and modify, we should keep the style the same, as if this document is done by a single person.

The above also applies for how the content is being written. For example:

> f. Techstack related:
    1. Any preferred frameworks?

Our questions are written in complete sentences, so this question is out of place. It should have been better phrased as something like:

> Are there any restrictions on the tech stack used for implementing the solution?

Lastly, I tend to type in a more casually and less refined over text, which should not be directly copied here. For e.g.

> Front-end: Are we supposed to host the app on app store? Or do we need an app, or just web will do?

I type in such a manner so that I don't spend that much time to phrase everything nicely, otherwise I would have made the changes and done the PR myself. So, this should be better phrased into something like:

> Should the digital solution be implemented in the form of a website or an app? In the case of an app, do we have to host the app on the app store ourselves?

**Comment by alvinlim540** [comment]
> Made a brief review. In summary, there are multiple English mistakes in the document. Ideally, we don't want to spend time reviewing each other's English, so put your content through a English checker first before submitting a PR.
> 
> Also, there are multiple stylistic issues e.g. unnecessary new lines, missing new line. To ensure that the content is easy to read and modify, we should keep the style the same, as if this document is done by a single person.
> 
> The above also applies for how the content is being written. For example:
> 
> > f. Techstack related:
> > 1. Any preferred frameworks?
> 
> Our questions are written in complete sentences, so this question is out of place. It should have been better phrased as something like:
> 
> > Are there any restrictions on the tech stack used for implementing the solution?
> 
> Lastly, I tend to type in a more casually and less refined over text, which should not be directly copied here. For e.g.
> 
> > Front-end: Are we supposed to host the app on app store? Or do we need an app, or just web will do?
> 
> I type in such a manner so that I don't spend that much time to phrase everything nicely, otherwise I would have made the changes and done the PR myself. So, this should be better phrased into something like:
> 
> > Should the digital solution be implemented in the form of a website or an app? In the case of an app, do we have to host the app on the app store ourselves?

I will take note of the points and further improve on crafting a better sentence. Thank you for the effort in correcting me.

**Pull request merged**